src/features/compensations/components/CreateCompensations/components/CompensationsTypesTips/CompensationsTypesTips.scss

```diff
@@ -0,0 +1,26 @@
+.compensations-types-tips {
+  margin-top: 16px;
+  width: 100%;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 1.4;
+  text-align: left;
```

Comment:
The default text-align value is start, which is equivalent to left. This property can be deleted.